docs/usage/config-overview.md

```diff
@@ -155,6 +155,12 @@ Inherited config may use all Repository config settings, and any Global config o
 
 For information on how the Mend Renovate App supports Inherited config, see the dedicated "Mend Renovate App Config" section toward the end of this page.
 
+#### Presets handling
+
+If presets are present in the inherited config, they will be resolved and added to the inherited config prior to being merged on top of global config.
```

Comment:
```suggestion
If the inherited config has presets, then Renovate:

1. Resolves the presets
1. Adds the resolved presets to the inherited config
1. Merges the presets on top of the global config
```

A numbered list of steps is easier than a long complicated sentence. Please check if this is technically accurate. 😉